client/jetpack-cloud/sections/partner-portal/assign-license-step-progress/index.tsx

```diff
@@ -44,21 +49,42 @@ interface Props {
 	currentStep: StepKey;
 	selectedSite?: SiteDetails | null;
 	showDownloadStep?: boolean;
+	isBundleLicensing?: boolean;
 }
 
-const AssignLicenseStepProgress = ( { currentStep, selectedSite, showDownloadStep }: Props ) => {
+const AssignLicenseStepProgress = ( {
+	currentStep,
+	selectedSite,
+	showDownloadStep,
+	isBundleLicensing,
+}: Props ) => {
 	const translate = useTranslate();
 	const paymentMethodRequired = useSelector( doesPartnerRequireAPaymentMethod );
 	const sites = useSelector( getSites ).length;
 
-	const steps: Step[] = [ { key: 'issueLicense', label: translate( 'Issue new license' ) } ];
+	const steps: Step[] = [
+		{
+			key: 'issueLicense',
+			label: isBundleLicensing ? translate( 'Select licenses' ) : translate( 'Issue new license' ),
+		},
+	];
+
+	if ( isBundleLicensing ) {
+		steps.push( {
+			key: 'reviewLicense',
+			label: translate( 'Review selections' ),
+		} );
+	}
 
 	if ( paymentMethodRequired ) {
 		steps.push( { key: 'addPaymentMethod', label: translate( 'Add Payment Method' ) } );
 	}
 
 	if ( sites > 0 && ! selectedSite ) {
-		steps.push( { key: 'assignLicense', label: translate( 'Assign license' ) } );
+		steps.push( {
+			key: 'assignLicense',
+			label: isBundleLicensing ? translate( 'Assign licenses' ) : translate( 'Assign license' ),
```

Comment:
Can we always default to the plural form here? In the current UI, it's possible to issue > 1 license as well, so I think it makes sense to use a consistent form. What do you think?

Reply:
Make sense. I agree to make this consistent 👍